changes.d/5721.feat.md

```diff
@@ -0,0 +1 @@
+Allow users to broadcast simulation mode settings to tasks.
```

Comment:
I don't think that's correct?? Run mode applies to the scheduler, not tasks

Reply:
In the past this has been true, but the point of this work is to make Cylc ready for Skip mode, and that requires the run mode to be applied on a task-by-task basis.

I think we should probably add the word "settings" - because these definately are changed on a task by task basis.